pandas-stubs/core/series.pyi

```diff
@@ -129,6 +129,13 @@ class _iLocIndexerSeries(_iLocIndexer, Generic[S1]):
     ) -> None: ...
 
 class _LocIndexerSeries(_LocIndexer, Generic[S1]):
+    # ignore needed because of mypy.  Overlapping, but we want to distinguish
+    # having a tuple of just scalars, versus tuples that include slices or Index
+    @overload
+    def __getitem__(  # type: ignore[misc]
+        self,
+        idx: Scalar | tuple[Scalar, ...],
+    ) -> S1: ...
```

Comment:
> but we want to distinguish having a tuple of just scalars, versus tuples that include slices or Index

Can you elaborate on which two cases this distunishes?

It might be nice to add inline comments which types/overloads are for index/multiindex.

I found two ways to use a tuple for a normal index, but I don't think this is an expected/typical usage:
```py
> type(pd.Series([1]).__getitem__((0, None)))
numpy.ndarray
> pd.Series([1], index=[("a", "b")]).__getitem__(("a", "b"))
1
```

Reply:
If you have a `MultiIndex`, then you can specify a tuple of index values to get a specific element.  You can use a tuple that contains a slice to get more than one value:
```python
>>> s = pd.Series([1,2,3,4], index=pd.MultiIndex.from_product([[1,2], ["a","b"]], names=["nn","ab"]))
>>> s
nn  ab
1   a     1
    b     2
2   a     3
    b     4
dtype: int64
>>> s.loc[1,"b"]
2
>>> s.loc[pd.IndexSlice[1,:]]
ab
a    1
b    2
dtype: int64
```

Reply:
next commit adds comments, and uses `_IndexSliceTuple`.  Note also that `s.loc[1,:]` works in example above.